mimecast/help.md

```diff
@@ -250,18 +252,18 @@ This action is used to add an email address or domain to a group.
 
 |Name|Type|Default|Required|Description|Enum|
 |----|----|-------|--------|-----------|----|
-|id|string|None|True|The Mimecast ID of the group to add to|None|
-|email_address|string|None|False|The email address of a user to add to a group. Use either email address or domain|None|
 |domain|string|None|False|A domain to add to a group. Use either email address or domain|None|
+|email_address|string|None|False|The email address of a user to add to a group. Use either email address or domain|None|
+|id|string|None|True|The Mimecast ID of the group to add to|None|
 
 ##### Output
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
-|folder_id|string|False|The Mimecast ID of the group that the user / domain was added to|
 |email_address|string|False|The email address of the user that was added to the group|
+|folder_id|string|False|The Mimecast ID of the group that the user / domain was added to|
 |id|string|False|The Mimecast ID of the user / domain that was added to the group|
-|internal|boolean|False|If the user / domain is internal or not|
+|internal|boolean|False|If the user / doamin is internal or not|
```

Comment:
@mhofert-r7 Can you start using `misspell` on every PR? It will catch this things so they don't make it in reviews.

Reply:
Ran it misspell says it's fine. It seems to miss a lot.

<img width="320" alt="Screen Shot 2019-12-12 at 1 18 45 PM" src="https://user-images.githubusercontent.com/32648900/70741890-fca34800-1ce1-11ea-8462-cf97ba3281ec.png">

Reply:
ah, fair call out. I thought it caught this instance but I can't reproduce.

Reply:
While it has a lot of false positives, `grammar.py` catches it.

```
$ cd mimecast
$ ../tools/grammar.py plugin.spec.yaml check
...
[FAIL] Possible spelling mistake found in sentence 'If the user / doamin is internal or not'. Possible suggestions: ['domain']
...
```